### PR TITLE
Translated using weblate (Malaysia)

### PR DESCRIPTION
**Pull request by Ondoz**

### CODE REVIEW

**Comment by ibennetch** [comment]
Thanks for your work on this. I'm a little confused about it, though; did you use the Weblate web page (at hosted.weblate.org) to edit this? Normally that tool merges changes directly instead of needing to open a pull request.

**Comment by Ondoz** [comment]
no i'm translite one by one  in my computer

**Comment by Ondoz** [comment]
if you don't mind i can move on

**Comment by williamdes** [comment]
> because I saw in the po folder for Malaysian there is no translation and if you don't mind i can move on thank's

Hello,
Please copy your work on https://hosted.weblate.org/languages/ms/phpmyadmin/
I am sorry not to be able to merge your work, this would make conflicts.
Files have to be modified my Weblate only :)

We will be happy to have some new translations.

File: po/ms.po [hunk 1]
```diff
@@ -272,6 +304,10 @@ msgid ""
 "refman/5.6/en/ha-memcached.html\">memcached</a> instead of the MySQL Query "
 "cache, especially if you have multiple slaves."
 msgstr ""
+"Anda menggunakan cache MySQL Query dengan pangkalan data trafik yang cukup tinggi."
+"mungkin perlu dipertimbangkan untuk menggunakan <a href = \" https://dev.mysql.com/doc/ "
+"refman / 5.6 / en / ha-memcached.html \"> memcached </a> dan bukannya MySQL Query "
```
Comment (williamdes) [inline po/ms.po:309]:
```suggestion
"mungkin perlu dipertimbangkan untuk menggunakan <a href=\"https://dev.mysql.com/doc/"
"refman/5.6/en/ha-memcached.html\"> memcached </a> dan bukannya MySQL Query"
```

File: po/ms.po [hunk 2]
```diff
@@ -106,17 +118,21 @@ msgid ""
...
 "environment. Usually a value of 1-5 seconds is suggested."
 msgstr ""
+"Dianjurkan untuk menetapkan {long_query_time} ke nilai yang lebih rendah, bergantung pada Anda"
+"persekitaran. Biasanya nilai 1-5 saat dicadangkan."
 
 #: libraries/advisory_rules_generic.php:76
 #, php-format
 msgid "long_query_time is currently set to %ds."
-msgstr ""
+msgstr "long_query_time kini ditetapkan kepada% ds."
```
Comment (williamdes) [inline po/ms.po:135]:
```suggestion
msgstr "long_query_time kini ditetapkan kepada %ds."
```

File: po/ms.po [hunk 3]
```diff
@@ -206,32 +230,37 @@ msgstr "Keterangan"
...
-msgstr ""
+msgstr "Manual MySQL hanya tepat untuk binari MySQL rasmi."
 
 #: libraries/advisory_rules_generic.php:163
 msgid ""
 "Percona documentation is at <a href=\"https://www.percona.com/software/"
 "documentation/\">https://www.percona.com/software/documentation/</a>"
 msgstr ""
+"Dokumentasi Percona ada di <a href = \" https://www.percona.com/software/ "
+"dokumentasi / \"> https://www.percona.com/software/documentation/ </a> "
```
Comment (williamdes) [inline po/ms.po:259]:
```suggestion
"Dokumentasi Percona ada di <a href=\"https://www.percona.com/software/"
"dokumentasi/\">https://www.percona.com/software/documentation/</a>"
```

**Comment by williamdes** [comment]
I think you did use an automated translator.

**Pull request closed without merging**